web/src/assets/styles/utilities.scss

```diff
@@ -105,3 +105,9 @@
   /** END block-size fallbacks **/
   block-size: 95dvh;
 }
+
+// fill all available space
+.fill {
+  block-size: 100%;
+  inline-size: 100%;
```

Comment:
I have been playing a bit with the Terminal and its Popup wrapper. To avoid the Popup scrollbars, you can

* To use a block size slightly smaller than 100%. `99%` or `calc(100% - 6px)` (not sure why 6px, do not have time now for further investigation), **or**
* To use the `.vertically-centered` CSS class instead. Looks like the CSS Grid manage the space better.

I'd use the second option by now.